backend/test_data.py

```diff
@@ -3,110 +3,110 @@
 import string
 from datetime import datetime
 
+
+def users(num, start=0):
+    '''
+    Generates a list of usernames for upvote/downvote lists
+    '''
+    return ['user' + str(i) for i in range(start, start+num)]
+
+
 # valid
 price0 = model.Price(
     user='admin',
-    upvote=10,
-    downvote=5,
+    upvotes=users(10),
+    downvotes=users(5, 10),
     price=1.99,
     date=datetime.now().timestamp()
 )
 
 # valid
 price1 = model.Price(
     user='admin',
-    upvote=7,
-    downvote=4,
+    upvotes=users(7),
+    downvotes=users(4, 7),
     price=0.99,
     date=datetime.now().timestamp()
 )
 
 # valid
 price2 = model.Price(
     user='admin',
-    upvote=0,
-    downvote=0,
+    upvotes=users(0),
+    downvotes=users(0, 0),
     price=0.00,
     date=datetime.now().timestamp()
 )
 
 # valid
 price3 = model.Price(
     user='admin',
-    upvote=200,
-    downvote=1000,
+    upvotes=users(200),
+    downvotes=users(1000, 200),
     price=139.99,
     date=datetime.now().timestamp()
 )
 
 # valid
 price4 = model.Price(
     user='admin',
-    upvote=20,
-    downvote=19,
+    upvotes=users(20),
+    downvotes=users(19, 20),
     price=12.89,
     date=datetime.now().timestamp()
 )
 
+# valid
+price9 = model.Price(
+    user='testuser',
+    upvotes=users(0),
+    downvotes=users(0, 0),
```

Comment:
"users(0)" and "users(0,0)" both return empty array correct? "[]" might be simpler to understand